mappings/net/minecraft/util/json/DynamicDeserializer.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_3817 net/minecraft/util/DynamicDeserializer
+CLASS net/minecraft/class_3817 net/minecraft/util/json/DynamicDeserializer
```

Comment:
This dynamic is not json at all; if you really want put it in `datafixer` package instead